src/test-extension-exemptions-workspace/test-extension-exemptions-workspace.service.ts

```diff
@@ -258,7 +268,7 @@ export class TestExtensionExemptionsWorkspaceService {
     record.pendingStatusCode = 'PENDING';
     record.submissionAvailabilityCode = 'REQUIRE';
 
-    await this.repository.save(record);
+    await repository.save(record);
     return this.getTestExtensionExemptionById(record.id);
```

Comment:
This should be updated to take the transaction as a parameter, and the method body should be updated to use it:
```suggestion
    return this.getTestExtensionExemptionById(record.id, trx);
```

Reply:
updated.